### PR TITLE
Crm-4094 remove secret squirrel deprecated package

### PR DESCRIPTION
**Pull request by wesonweb**
## Why? 

This PR is one of the steps in [JIRA 4094](https://financialtimes.atlassian.net/browse/CRM-4094) to remove deprecated packages and apply patches.

Secret Squirrel is deprecated and can be removed from this system as the FT uses Github Advanced Security.

## Context

This follows on from [PR112 ](https://github.com/Financial-Times/tps-lookup/pull/112) which removed Snyk and NodeJsScan from the system. 

## What has changed

This PR:
- uninstalls Secret Squirrel and removes its related file 
- updates the package.json file to remove the script that runs Secret Squirrel
- updates the `Readme` file with the correct Splunk search query

## Testing

- App deploys to Heroku so will check Heroku logs and Splunk query to ensure no issues
- Will perform a 'before and after' TPS lookup to ensure the expected status is returned for phone numbers.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,14 +43,7 @@
     }
   },
   "devDependencies": {
-    "@financial-times/secret-squirrel": "^2.17.0",
     "husky": "^1.3.1",
```
Comment (wesonweb) [inline package.json:46]: I thought Husky was only being used for secret squirrel so removed it but reinstalled it as the system is still looking for it to be used, even though it's not referenced in the codebase.

File: README.md [hunk 2]
```diff
@@ -34,7 +34,11 @@ Logging for the `updateNumber.js` file is sent to Splunk from Heroku. Functions
 
 See the Splunk query below:
 
-`index="heroku" source="/var/log/apps/heroku/ft-ft-tps-screener.log"`
```
Comment (wesonweb) [inline README.md:37]: I could not find any Splunk logs using this query but was able to access success and error logs using the new query inserted below.

**Comment by wesonweb** [comment]
Update: `tps-lookup` has been removed as a project from Snyk

File: package.json (hunk 1 above)
Comment (nl-ria) [reply package.json:46]: How do you mean? Having had a look, it does seem Husky was running only `secret-squirrel` as a precommit. Are you seeing some issues after uninstalling husky and secret-squirrel?

Comment (nl-ria) [reply package.json]: How did you uninstall husky Wes? I just tried the following : 

`npm uninstall husky`

and the output was: 
```                                  
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: 'tps-lookup@1.0.0',
npm warn EBADENGINE   required: { node: '18.x' },
npm warn EBADENGINE   current: { node: 'v20.12.1', npm: '10.8.3' }
npm warn EBADENGINE }
npm warn EBADENGINE Unsupported engine {
npm warn EBADENGINE   package: '@okta/oidc-middleware@5.1.0',
npm warn EBADENGINE   required: { node: '^12.19.0 || ^14.15.0 || ^16.13.0 || ^18.14.0' },
npm warn EBADENGINE   current: { node: 'v20.12.1', npm: '10.8.3' }
npm warn EBADENGINE }
npm warn deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm warn deprecated csurf@1.11.0: Please use another csrf package
npm warn deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm warn deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm warn deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm warn deprecated w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
npm warn deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
npm warn deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm warn deprecated text-encoding@0.7.0: no longer maintained

added 1006 packages, and audited 1007 packages in 29s

80 packages are looking for funding
  run `npm fund` for details

50 vulnerabilities (3 low, 29 moderate, 13 high, 5 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
```
I was then successfully able to commit the changes in this [commit]([Retry committing after removing husky](https://github.com/Financial-Times/tps-lookup/pull/114/commits/d52a3fb15f6f8b21d96e1746e4fed4e3cd7d4ea5)) without any error or warning

**Review by nl-ria — APPROVED** [review]
This looks good to me!

**Comment by nl-ria** [comment]
As an FYI - I have also closed the unmerged Snyk PRs from this repo since they were more than a month old. Once we enable security upgrades in this repo, dependabot will open fix PRs in the future that we can address.

File: package.json (hunk 1 above)
Comment (wesonweb) [reply package.json]: How strange - I used `npm uninstall husky` to remove it but got that message `Can't find Husky` after do so. Thank you for sorting it :-)  👍

File: README.md (hunk 2 above)
Comment (wesonweb) [reply README.md:37]: A separate logging issue was resolved in [this PR](https://github.com/Financial-Times/tps-lookup/pull/115).

**Pull request merged**